chgnet/model/dynamics.py

```diff
@@ -33,6 +33,18 @@
     from ase.io import Trajectory
     from ase.optimize.optimize import Optimizer
 
+try:
+    from ase.filters import Filter, FrechetCellFilter
+except ImportError:
+    print(
+        "We recommend using ase's unreleased FrechetCellFilter over ExpCellFilter for "
+        "CHGNet structural relaxation. ExpCellFilter has a bug in its calculation "
+        "of cell gradients which was fixed in FrechetCellFilter. Otherwise the two "
+        "are identical. ExpCellFilter was kept only for backwards compatibility and "
+        "should no longer be used. Run pip install git+https://gitlab.com/ase/ase to "
+        "install from main branch."
+    )
+
```

Comment:
@BowenD-UCB We now print this advice when `ase.filters.FrechetCellFilter` can't be imported.

Reply:
thanks for this!